inc/shortcodes.php

```diff
@@ -212,44 +215,76 @@ function sell_media_checkout_shortcode() {
 							<?php echo $item['item_size']; ?>
 						<?php endif; ?>
 						</div>
-						<div class="item-license">
-						<?php if ( ! empty( $item['item_usage'] ) ) : ?>
-							<?php echo $item['item_usage']; ?>
+						<?php
+						$markups = Sell_Media()->tax_markup->markup_taxonomies();
+						if ( is_array( $markups ) && count( $markups ) > 0 ) : ?>
+							<?php foreach ( $markups as $markup ) : ?>
+								<?php if ( 'licenses' === $markup ) : ?>
+									<div class="item-license">
+										<?php if ( ! empty( $item['item_usage'] ) ) : ?>
+											<?php echo $item['item_usage']; ?>
+										<?php endif; ?>
+									</div>
+								<?php else : ?>
+									<div class="item-{$markup}">
+										<?php if ( ! empty( $item[ "item_markup_{$markup}" ] ) ) : ?>
+											<?php echo $item[ "item_markup_{$markup}" ]; ?>
+										<?php endif; ?>
+									</div>
+								<?php endif; ?>
+							<?php endforeach; ?>
 						<?php endif; ?>
-						</div>
 					</div>
-					<div class="item-qty-total">
-						<div class="item-decrement">
-							<span class="sell-media-cart-decrement dashicons dashicons-minus"></span>
-						</div>
+						
 						<div class="item-quantity">
 							<span class="count">
 								<?php if ( ! empty( $item['qty'] ) ) : ?>
 									<?php echo $item['qty']; ?>
 								<?php endif; ?>
 							</span>
 						</div>
-						<div class="item-increment">
-							<span class="sell-media-cart-increment dashicons dashicons-plus"></span>
+						
+						<div class="item-total" style="margin-left:-6px;">
+						<?php echo sell_media_get_currency_symbol( $settings->currency ) . number_format( $item['price'] * $item['qty'], 2 ); ?>
 						</div>
-						<div class="item-total">
-							<?php echo sell_media_get_currency_symbol( $settings->currency ) . number_format( $item['price'] * $item['qty'], 2 ); ?>
+						<div class="item-decrement remove-item">
+							<span class="hide-xs">
+							<?php echo sell_media_get_currency_symbol( $settings->currency ) . number_format( $item['price'] * 
+							$item['qty'], 2 ); ?></span>
+							<span class="sell-media-cart-decrement dashicons  dashicons-no-alt"></span>
 						</div>
-					</div>
+						
+						
 				</li>
 			<?php
 			$cart_index++;
 			endforeach; ?>
 		</ul>
 
 		<?php do_action( 'sell_media_checkout_after_cart' ); ?>
+	</div>
 
+<div class="right-sell-coulmn">
+		
+	
+		<?php do_action( 'sell_media_checkout_after_cart' ); ?>
+		
 		<div class="sell-media-totals group">
+
+			<div class="promorcode-sell">
```

Comment:
Promo codes are added with an extension. Please remove all of the fields  you added for promo codes and the CSS.

Reply:
Removed, I was thinking if you can send me that extension so that i can check the layout it adds